ui/app/AppLayouts/Browser/components/BookmarkButton.qml

```diff
@@ -0,0 +1,46 @@
+import QtQuick 2.13
+import "../../../../shared"
+import "../../../../shared/status"
+import "../../../../imports"
+
+Item {
+    property url source
+    property string text
+    signal clicked(mouse: var)
+    signal rightClicked(mouse: var)
+
+    id: root
+    width: 74
+    height: bookmarkImage.height + bookmarkName.height + Style.current.halfPadding
```

Comment:
```suggestion
    height: childrenRect.height + Style.current.halfPadding
```

Reply:
I can't do that sadly because I have a MouseArea that fills the parent so it would create a binding loop